scipy/signal/signaltools.py

```diff
@@ -465,9 +517,9 @@ def convolve(in1, in2, mode='full'):
     slice_obj = [slice(None, None, -1)] * len(kernel.shape)
 
     if np.iscomplexobj(kernel):
-        return correlate(volume, kernel[slice_obj].conj(), mode)
```

Comment:
Calling .conj() on real input takes 100 ns on my machine, so I think the if/then can be removed for simplicity?